browser/src/control/Control.UIManager.js

```diff
@@ -511,6 +514,131 @@ L.Control.UIManager = L.Control.extend({
 		}
 	},
 
+	initializeNavigator: function (docType) {
```

Comment:
can we move this function inside `navigator` file?
then here just call   ` this.navigator.initialize(...);`

also remember inside navigator instances of the containers as memeber variables on initialize.
we do not need to query them all the time